### ISSUE TITLE
Optional samba configuration

### ISSUE THREAD

**Issue by Tuxem** [Thulium-Drake/ansible-role-adjoin#8]
Samba configuration is not mandatory to join the domain

**Comment by Thulium-Drake** [comment]
I don't really see how Samba is optional, as I use the net command from Samba to do the actual AD join.

How did you do it without (note: I'm not a Windows expert, I just made it work ;-) )

**Comment by Tuxem** [comment]
There is a much simpler way with SSSD + realmd.  Less configuration needed.
I'll try to fork and make a little refacto and you'll see if it's convenient for you.

**Comment by Thulium-Drake** [comment]
Fair enough, it's been quite some time since I initially wrote this. I don't remember if we even could use realmd back then.

Anyway, I never really bothered to change it, as it was still working fine for where I needed it for :)
